### ISSUE TITLE
API Bulk update for users and groups

### ISSUE THREAD

**Issue by boesr** [goauthentik/authentik#10694]
**Is your feature request related to a problem? Please describe.**
I integrated the Authentik API in one of our tools. In one use case I need to update a attribute of all users that have a certain value of that attribute and are in a defined group. In my test scenarios these are around 1000 users. When trying to update them in a loop the ram usage explodes and finally breaks authentik (tested with 48 GB RAM). For now I increased the RAM and will probably add some sleeps between the updates, but it already is quiet slow in comparison to a direct data base update.

**Describe the solution you'd like**
It would be nice to have a PUT route to update attribute values of all users / groups with a certain value for a defined attribute.

**Describe alternatives you've considered**
I considered doing it directly within the database via a SQL query. Something like

```
UPDATE authentik_core_user
   set attributes['my_attribute'] = to_jsonb(15);
WHERE
  attributes->>'my_attribute' = 14
```

**Additional context**
/

**Comment by boesr** [comment]
The above error only is thrown when I execute nearly in parallel. I adjusted to do it sequentially and now everything seems to work although it could be faster.
